python/rikai/contrib/torch/__init__.py

```diff
@@ -11,3 +11,16 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
+
+from abc import ABC, abstractmethod
+from typing import Any, Dict
+
+
+class RikaiModule(ABC):
```

Comment:
Do you intent to store this model into model registry? Will this interface break other workload, i.e., model deployment in non-rikai enrionment?

Reply:
> Do you intent to store this model into model registry?
Yes

> Will this interface break other workload
No

Because the model logged for Rikai (batch processing purpose) can be separated from the model for serving and other use cases.

Reply:
> PickleError: ScriptModules cannot be deepcopied using copy.deepcopy or saved using torch.save. Mixed serialization of script and non-script modules is not supported. For purely script modules use my_script_module.save(<filename>) instead.

```
import torch.nn as nn
import torch

class WrappedModel(nn.Module):
    def __init__(self, model):
        super().__init__()
        self.model = model.eval()

    @torch.no_grad()
    def forward(self, imgs):
        preds = self.model(imgs)
        return (preds, imgs)
```

I tried to log the wrapped model to MLflow, but failed, because it is a torch script model.

Reply:
There are ways to use the wrapped model without logging it to MLflow (or model store).

As discussed in https://github.com/eto-ai/rikai/issues/354